tests/app/core/models/github/github-label.model.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { GithubLabel } from '../../../../../src/app/core/models/github/github-label.model';
+
+describe('GithubLabel', () => {
+  let githubLabel: GithubLabel;
+  const category = 'severity';
+  const value = 'LOW';
+
+  it('should returns false on isCategorical() when provided wit no name', () => {
+    githubLabel = new GithubLabel({ });
+    expect(githubLabel.isCategorical()).toEqual(false);
+  });
+
+  it('should returns false on isCategorical() when provided with invalid label name', () => {
+    githubLabel = new GithubLabel({ name: `${category}${value}`  });
+    expect(githubLabel.isCategorical()).toEqual(false);
+
+    githubLabel = new GithubLabel({ name: `${category}             ${value}`  });
+    expect(githubLabel.isCategorical()).toEqual(false);
+  });
+
+  it('should returns true on isCategorical() when provided with valid label name', () => {
```

Comment:
Let's change the spec description to `... when provided with categorical label name`.

Reply:
:+1:

Reply:
I think you missed this

Reply:
Okays!

Reply:
Should have fixed the tests naming convention already!